<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_ui_message">
    <sys_ui_message action="INSERT_OR_UPDATE">
        <application/>
        <code/>
        <commented_message/>
        <key>received a '{3}' event on {0} number {1} ({2}) with call id {4} and status '{6}' (get next: {5})</key>
        <language>uk</language>
        <message>отримано подію «{3}» на номер {0} {1} ({2}) з ідентифікатором виклику {4} та станом {6} (отримати наступне: {5})</message>
        <sys_class_name>sys_ui_message</sys_class_name>
        <sys_created_by>bogdan.felinskyi</sys_created_by>
        <sys_created_on>2022-03-23 17:39:47</sys_created_on>
        <sys_id>e15f342b1b96c910c069657bab4bcb6e</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>received a '{3}' event on {0} number {1} ({2}) with call id {4} and status '{6}' (get next: {5})</sys_name>
        <sys_package display_value="I18N: Ukrainian Translations" source="176ebda287b38910f4b131570cbb3551">176ebda287b38910f4b131570cbb3551</sys_package>
        <sys_policy/>
        <sys_scope display_value="I18N: Ukrainian Translations">176ebda287b38910f4b131570cbb3551</sys_scope>
        <sys_update_name>sys_ui_message_e15f342b1b96c910c069657bab4bcb6e</sys_update_name>
        <sys_updated_by>bogdan.felinskyi</sys_updated_by>
        <sys_updated_on>2022-03-23 17:39:47</sys_updated_on>
    </sys_ui_message>
</record_update>
